app/templates/pwa/default/custom-service-worker.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-env serviceworker */
```

Comment:
```suggestion
```

This is no longer required as the new ESLint config override sets this. So, we can get rid of this.

Reply:
This is only on the JS version, since I didn't put the ESLint override there (it shouldn't be needed)